pages/Configuring/Configuring-Hyprland.md

```diff
@@ -38,14 +39,14 @@ COMMAND = VALUE
 
 is valid.
 
-# Basic configuring
+## Basic configuring
```

Comment:
again why, looks awful in the toc

Reply:
Wouldn't know, CAN'T PREVIEW MY CHANGES.

Reply:
unlucky